docs/api.md

```diff
@@ -58,6 +58,7 @@
     + [page.press(key[, options])](#pagepresskey-options)
     + [page.reload(options)](#pagereloadoptions)
     + [page.screenshot([options])](#pagescreenshotoptions)
+    + [page.setBlockedURLs(urls)](#pagesetblockedURLs)
```

Comment:
it seems this should be `#pagesetblockedurlsurls`

Reply:
Fixed, next time I will check what is the generated anchor :)